source/auth/auth.rst

```diff
@@ -159,8 +159,11 @@ handshake:
    MongoDB Handshake fails with an error, drivers MUST treat this as
    an authentication error.
 
-#. If the server is not of type Standalone, RSPrimary, RSSecondary or
-   Mongos, no authentication is possible and the handshake is complete.
+#. If the server is of type Standalone, RSPrimary, RSSecondary or Mongos,
+   continue with authentication in Step 3. If the connection is a direct
+   connection and the server is not type RSArbiter, continue with
+   authentication in Step 3. Otherwise, no authentication is possible and the
+   handshake is complete.
```

Comment:
Would specifying a "single topology type" be preferable to "direct connection" in this text?

SDAM requires drivers to support [direct connections](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#general-requirements), but [Initial TopologyType](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#initial-topologytype) suggests that the behavior corresponds to forcing a single topology type:

> The user MUST be able to set the initial TopologyType to Single.

This came up recently in the scope for [WRITING-4391](https://jira.mongodb.org/browse/WRITING-4391) (unify replica set discovery behavior) and we agreed that to be strict about referring to direct connections as having a single topology type.

Reply:
Updated.